__tests__/app.test.js

```diff
@@ -303,4 +303,26 @@ describe('GET /api/users', ()=>{
             expect(body.msg).toBe("Route not found")
         })
     })
+})
+
+describe('GET /api/articles?topic=cats', ()=>{
+    test('should filter responses based on the topic added in the query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles.length).toBe(1)
+        })
```

Comment:
This does not actually prove that we are getting articles back that are filtered by the topic of cats. There should be an expect statement ensuring that every article has all of the expected properties including topic which in this case should have the value of cats